### ISSUE TITLE
when I use the pre-trained pose model to predict, the output is same whatever input are

### ISSUE THREAD

**Issue by KerunLi** [tinghuiz/SfMLearner#135]
when I use the pre-trained pose model to predict, the output is same whatever inputs are.

I use seq 9 and seq 11 to run test_kitti_pose.py, but **the output pose.txt are all the same**, that is, 000000.txt and 000001.txt has the same content.

I print the input image data, it's not same, but the output pose is same. I normalized the input image data but it doesn't work.

（I use tensorflow-gpu 2.0.0b, cuda 10.0 and python 3.6, and use "`import tensorflow.compat.v1 as tf`" and "`tf.disable_v2_behavior()`" replace `import tensorflow as tf`

thanks a lot for any help.

**Comment by Yifan-github** [comment]
i have a question.where is the kitti_depth_model.tar?

i can't download the pre-trained model by running the "bash ./models/download_depth_model.sh".

Hope to get a help,thanks a lot.

**Comment by KerunLi** [comment]
@Yifan-github You can try run `wget http://people.eecs.berkeley.edu/~tinghuiz/projects/SfMLearner/models/kitti_depth_model.tar` in terminal, then unpack it.
if you still cann't download, maybe it's the problem of network or vpn

**Comment by Yifan-github** [comment]
> 当我使用预训练的姿势模型进行预测时，无论输入是什么，输出都是相同的。
> 
> 我用seq 9和seq 11运行test_kitti_pose.py，但是**输出的pose.txt都是一样的**，即000000.txt和000001.txt的内容是一样的。
> 
> 我打印输入图像数据，它不一样，但输出姿势是一样的。我对输入图像数据进行了标准化，但它不起作用。
> 
> （我使用tensorflow-gpu 2.0.0b、cuda 10.0和python 3.6，并用“ `import tensorflow.compat.v1 as tf`”和“ `tf.disable_v2_behavior()`”替换`import tensorflow as tf`
> 
> 非常感谢您的帮助。



> @Yifan-github您可以尝试`wget http://people.eecs.berkeley.edu/~tinghuiz/projects/SfMLearner/models/kitti_depth_model.tar`在终端中运行，然后解压。
> 如果还是不能下载，可能是网络或者vpn的问题

**Comment by Yifan-github** [comment]
i met a ERROR 404: Not Found.Is the file sharing cancelled?If possible, may you share me a copy by email?Thanks so much!My E-mail:820407324@qq.com
